### ISSUE TITLE
We have a number of security issues in salior

### ISSUE THREAD

**Issue by zubairov** [elasticio/sailor-nodejs#59]
See the ``nsp`` report:

```
(+) 4 vulnerabilities found
 Name      Installed   Patched                        Path                                                                 More Info                              
 debug     2.1.3       >= 2.6.9 < 3.0.0 || >= 3.1.0   elasticio-sailor-nodejs@2.0.0-dev.10 > debug@2.1.3                   https://nodesecurity.io/advisories/534 
 ms        0.7.0       >0.7.0                         elasticio-sailor-nodejs@2.0.0-dev.10 > debug@2.1.3 > ms@0.7.0        https://nodesecurity.io/advisories/46  
 request   2.57.0      >=2.68.0                       elasticio-sailor-nodejs@2.0.0-dev.10 > request@2.57.0                https://nodesecurity.io/advisories/309 
 hawk      2.3.1       >=3.1.3 < 4.0.0 || >=4.1.1     elasticio-sailor-nodejs@2.0.0-dev.10 > request@2.57.0 > hawk@2.3.1   https://nodesecurity.io/advisories/77  
```

**Comment by zubairov** [comment]
should be fixed in #60

**Comment by jhorbulyk** [comment]
The fix for this issue has been published.
